### PR TITLE
AC-308: Update README on GitHub

### PR DESCRIPTION
**Pull request by ykarim**
Updated information and added screenshots.

### CODE REVIEW

**Comment by shivtej1505** [comment]
@ykarim , it's looking good :)

**Comment by ykarim** [comment]
Thanks, @shivtej1505 :)

File: README.md [hunk 1]
```diff
@@ -1,78 +1,101 @@
...
 # Objectives
-### Version 2.0.1
-- Starting/stopping visits
-- To Be Determined
-[See more in JIRA](https://issues.openmrs.org/browse/AC/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel)
+### Version 2.6 (next release)
+### Goals: 
+1. Add/Edit Patient Pictures
+2. Lint Code Review Integration
+3. Asynchronous Data Sycnhronization
```
Comment (AdamGrzybkowski) [inline README.md]: @ykarim I would add here patient photo feature

File: README.md [hunk 2]
```diff
@@ -1,78 +1,101 @@
...
-- To start development on local server, type http://10.0.2.2:8080/openmrs (for Android Studio AVD) or http://10.0.3.2:8080/openmrs (for Genymotion) as URL.
+##### Demo Username : admin
+##### Demo Password : Admin123
 
 # Releasing
 
-In order to release the application, go to [releases](https://github.com/openmrs/openmrs-contrib-android-client/releases) and click the [Draft a new release](https://github.com/openmrs/openmrs-contrib-android-client/releases/new) button. It will create a new version tag in the repository and build the app. The tag name will be used as the version number for this release.
-
-If you want to release a new major or minor version, please be sure to update the applicationVersion variable in [build.gradle](https://github.com/openmrs/openmrs-contrib-android-client/blob/master/openmrs-client/build.gradle#L26) prior to the release.
+We follow the sprint model for development. Read more about it here: [OpenMRS Sprints](https://wiki.openmrs.org/display/RES/Development+Sprints).
```
Comment (AdamGrzybkowski) [inline README.md]: I'm not sure about deleting the instructions. Maybe add them below ?

File: README.md (hunk 1 above)
Comment (ykarim) [inline README.md]: @AdamGrzybkowski I've added it at the top. Shall I move it to the bottom?

**Comment by ykarim** [comment]
@AdamGrzybkowski Updated PR. :)

File: README.md (hunk 1 above)
Comment (AdamGrzybkowski) [inline README.md]: @ykarim oh sorry... I've totally missed that :man_facepalming:

**Pull request merged**